pkgs/by-name/th/thc-secure-delete/package.nix

```diff
@@ -4,35 +4,39 @@
 , pkg-config
 }:
 
+let
+  version = "3.1.1";
```

Comment:
There's no need to have a `let...in` clause, use `finalAttrs.version` when you need the `version`.

Reply:
I introduced the let...in because it keeps on resolving to mirror not found error.

Reply:
... ?